README.md

```diff
@@ -47,7 +47,7 @@ as ``ovirt_engine_setup_answer_file_path``.
 |---------------------------------|-----------------------|-----------------------------------------------------------|
 | ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
 | ovirt_engine_setup_use_remote_answer_file | False             | If `True`, use answerfile's path on the remote machine. This option should be used if the installation occure on the remote machine and the answerfile is located on there also. |
-| ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
+| ovirt_engine_setup_update_setup_packages | True              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
```

Comment:
Wouldn't it cause some issue during new installation?

Reply:
I would not think so, but can't be sure until we try it. If yes, we should enforce False on fresh install, and use this var only in upgrade.

Reply:
Affected part of code:

  - name: Update setup packages
    # WA for https://github.com/ansible/ansible/issues/34867 whic didn't get
    # fixed by suggested update_only parameter.
    command: yum -y update ovirt\*setup\*
    when: ovirt_engine_setup_update_setup_packages or ovirt_engine_setup_perform_upgrade
    args:
      warn: false
    tags:
      - "skip_ansible_lint"  # ANSIBLE0006


Thus should not affect, which brings me to thinking on when condition that this is completely useless.

Reply:
Right, we change implementation to use yum module instead of calling yum command directly, because about issue was fixed in 2.7.3 and we require 2.9.0

Martin could you please post separate PR for this?
@mnecas